resources/js/types/components.d.ts

```diff
@@ -12,7 +12,7 @@ declare module "vue" {
     AppMessage: typeof import("./../components/app/Message.vue")["default"];
     AppNavbar: typeof import("./../components/app/Navbar.vue")["default"];
     BoxContainer: typeof import("./../components/box/Container.vue")["default"];
-    CardAvatar: typeof import("./../components/card/Avatar.vue")["default"];
+    CardAvatarCard: typeof import("./../components/card/AvatarCard.vue")["default"];
```

Comment:
Did you regenerate the file or why is the line still there? Could you test with `npm run dev` and then lint via. `npm run lint:fix
./vendor/bin/pint`

Reply:
I merge you code in my branch, and so this good fixed automatically